test/JDBC/input/BABEL-2144-vu-verify.sql

```diff
@@ -0,0 +1,18 @@
+UPDATE babel_2144_vu_prepare_t1 SET c1 = 23 OUTPUT INSERTED.c2, 22 INTO babel_2144_vu_prepare_v1;
```

Comment:
UPDATE is not the only query where OUTPUT INTO clause can be used, it can used in INSERT and DELETE statements as well. So we need to add test cases for these as well.